ExampleService_WebApi - Copy/obj/ExampleService_WebApi.csproj.nuget.dgspec.json

```diff
@@ -0,0 +1,77 @@
+{
```

Comment:
We should not commit the binary files. Like the auto generated files inside `obj` and `bin` folders. 

We should add [.gitignore file](https://git-scm.com/docs/gitignore#:~:text=A%20gitignore%20file,specifies%20a%20pattern) to tell git to not to consider these files as changes.

Reply:
# Implementation
Add git ignore file